java-extensions/jdbc-bridge/src/main/java/com/starrocks/jdbcbridge/DataSourceCache.java

```diff
@@ -0,0 +1,27 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.jdbcbridge;
+
+import com.zaxxer.hikari.HikariDataSource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// a cache for get datasource
+public class DataSourceCache {
+    private final Map<String, HikariDataSource> sources = new HashMap<>();
```

Comment:
Do we need to limit the cache size?

Reply:
I think it's unecessary. Each JDBC resource has it's connection pool, and multiple tables will share a single connection pool